osu.Server.Queues.ScoreStatisticsProcessor.Tests/DatabaseTest.cs

```diff
@@ -32,7 +31,7 @@ public abstract class DatabaseTest : IDisposable
 
         protected const int TEST_BEATMAP_ID = 1;
         protected const int TEST_BEATMAP_SET_ID = 1;
-        protected int TestBuildID;
+        protected ushort TestBuildID;

```

Comment:
Technically `uint` is better (column is `mediumint unsigned`) but sure.

Reply:
column is `smallint` in the new table!